Demo/Demo/SceneDelegate.swift

```diff
@@ -23,4 +23,56 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     }
 }
 
-extension SceneDelegate: TurboNavigationDelegate {}
+import WebKit
+
+/// This example class shows how one can use more features of Turbo Navigator.
+class ComplexSceneDelegate: UIResponder, UIWindowSceneDelegate {
+    var window: UIWindow?
+
+    private lazy var turboNavigator = TurboNavigator(pathConfiguration: pathConfiguration, delegate: self)
+    private let pathConfiguration = PathConfiguration(sources: [
+        .server(baseURL.appendingPathComponent("/configurations/ios_v1.json"))
+    ])
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        guard let windowScene = scene as? UIWindowScene else { return }
+
+        TurboConfig.shared.userAgent += " CUSTOM STRADA USER AGENT"
```

Comment:
Definitely something to consider later, but I think this is the last piece of the puzzle that I can't make fit. I don't like the static configuration. I know Android has a WebView subclass due to technical constraints. Maybe we can make it work somehow.

Reply:
Agreed. This static configuration made more sense (to me) before Strada existed and I was only setting the user agent. Honestly, we could even rip this out of turbo-ios (i.e. never upstream it) and force folks to use the `Session` initializer if they want to customize the web view. But I'll see how that plays out in the upstream PR.

Reply:
Note that we have an instance in HEY where our composer's WebView is separate from the rest of the app and does not support the same Strada components as the rest of the app, so the user-agent is actually different for that webview. So we should support continue to support situations where the apps can set the user agent per `WebView/Session` — but that doesn't need to be the default path for simpler apps.

On the Android side, we just notify the app see whenever the session is (re)created and can update the user agent directly for the `webView`: https://github.com/hotwired/turbo-android/blob/main/demo/src/main/kotlin/dev/hotwire/turbo/demo/main/MainSessionNavHostFragment.kt#L49

Reply:
Agreed. Both the user agent and the web view subclass should be supported. I was just saying there should be a better way of achieving both goals than a static configuration.